### ISSUE TITLE
Paperclip deprecated

### ISSUE THREAD

**Issue by LocoDelAssembly** [SpeciesFileGroup/taxonworks#488]
Thoughbot announced Paperclip is deprecated: https://robots.thoughtbot.com/closing-the-trombone

They provide a migration guide to ActiveStorage: https://github.com/thoughtbot/paperclip/blob/master/MIGRATING.md

What it looks interesting to me of ActiveStorage (although not sure if could be done with Paperclip) is the file path generation, kinda UUID, which would make easier to deploy imported images to servers without the risk of collision (just re-importing with `Image.new image_file: ...` would be very time consuming as the scaled images would need to be regenerated)

**Comment by mjy** [comment]
Yes, saw this a couple months back.  We'll wait till we get to the 5.2
branch to deal with it I think.
